src/main.go

```diff
@@ -65,6 +65,11 @@ func main() {
 
 	fmt.Println("Bot is open and listening...")
 
+	dbHandler := infrastructure.NewSqliteHandler("/var/tmp/production.sqlite")
```

Comment:
Remove this string. Put it in an environment variable and access it via godotenv